hikari/guilds.py

```diff
@@ -851,7 +1132,7 @@ class WelcomeScreen:
 
 @attr_extensions.with_copy
 @attr.define(hash=False, kw_only=True, weakref_slot=False)
-class GuildMemberBan:
+class Ban:
```

Comment:
Maybe best to rename to `GuildBan` instead.

Reply:
Ah, I will change that.